src/main/resources/graphql/hentPerson.gql

```diff
@@ -45,6 +45,9 @@ query($ident: ID!) {
         ...metadataDetails
       }
     }
+    adressebeskyttelse {
```

Comment:
retrive metadata here: ...metadataDetails

Reply:
Fixed, thanks for noticing this!